src/js/renderers/TextControl.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { withJsonFormsControlProps } from "@jsonforms/react";
+import { rankWith, isStringControl } from "@jsonforms/core";
+import { TextControl as UiTextControl } from '@wordpress/components';
+import { useState } from '@wordpress/element';
+
+const TextControl = (props) => {
+  const {
+    id,
+    description,
+    errors,
+    label,
+    uischema,
+    path,
+    visible,
+    required,
+    config,
+    data,
+    input,
+    handleChange
+  } = props;
+  
+  return ( 
+    <UiTextControl
+      help={description}
+      label={label}
+      value={data || ''}
+      onChange={(value) =>
```

Comment:
According to [Gutenberg document](https://wordpress.github.io/gutenberg/?path=/docs/components-textcontrol--with-label-and-help-text), the input's value can be parsed as the first parameter